scripts/setup_qemu_optee.sh

```diff
@@ -254,6 +305,26 @@ $DEV_PATH/build_linux.sh
 # Save kernel version for later use
 export KERNEL_VERSION=`cd $DST_KERNEL && make kernelversion`
 
+
+################################################################################
+# Generate the build script for SE API test                                    #
+################################################################################
+if [ -n "$WITH_SE_API_TEST" ]; then
+
+cat > ${DEV_PATH}/build_se_api_test.sh << EOF
+#!/bin/sh
+
```

Comment:
We also need to export the toolchain in use, I think this is the line missing
export PATH=${DEV_PATH}/toolchains/aarch32/bin:$PATH